srcpkgs/sing-geosite/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'sing-geosite'
+pkgname=sing-geosite
+version=20240713050854
```

Comment:
This version breaks our version scheme, we don't support this large version!

Reply:
I wasn't aware that this version would cause a break, so I went ahead and changed the version of the file.